mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -184,12 +166,19 @@ object CaseKeywordCompletion:
    * @param typedtree typed tree of the file, used for generating auto imports
    * @param indexedContext
    * @param config
+   * @param parent the parent tree node of the pattern match, for example `Apply(_, _)` when in
+   *               `List(1).foreach { cas@@ }`, used as fallback to compute the type of the selector when
+   *               it's `EmptyTree`.
+   * @param casePrefix when in `List(1).foreach { case@@ }` we want user to see `case (exhaustive) Option (2 cases)`,
+   *                   when false (default) we get `match (exhaustive) Option (2 cases)`
```

Comment:
`indexedContext` and `config` can be removed from scaladoc as there is no exlanation for them, and as they're more or less obvious.
Although, I have to admit that `IndexedContext` on its own doesn't have any scaladoc which is a tiny obstacle for person who isn't familiar with that code. Fancy to add some?

Reply:
I'm not sure if I'm able to write a proper explanation of `IndexedContext` 😅  but I can try. Let's do it in a different PR tho